project-base/app/MultiDomainEntityClassProvider.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Shopsys;
+
+use Shopsys\FrameworkBundle\Component\Domain\Multidomain\MultiDomainEntityClassProviderInterface;
+use Shopsys\FrameworkBundle\Component\Setting\SettingValue;
+use Shopsys\FrameworkBundle\Model\Category\CategoryDomain;
+use Shopsys\FrameworkBundle\Model\Mail\MailTemplate;
+use Shopsys\FrameworkBundle\Model\Payment\PaymentDomain;
+use Shopsys\FrameworkBundle\Model\Product\Brand\BrandDomain;
+use Shopsys\FrameworkBundle\Model\Product\ProductDomain;
+use Shopsys\FrameworkBundle\Model\Product\ProductVisibility;
+use Shopsys\FrameworkBundle\Model\Transport\TransportDomain;
+
+class MultiDomainEntityClassProvider implements MultiDomainEntityClassProviderInterface
+{
+    /**
+     * @return string[]
+     */
+    public function getIgnoredMultidomainEntitiesNames(): array
```

Comment:
your document states that the implementation should be in framework package. Or am I getting it wrong? Are you sure that those definitions belong into project base and not into framework?

Reply:
Well. I think that this should be in project-base, because there would be 3 implementations of MultiDomainEntityClassProviderInterface. One in component, second in framework, and for project-base there, would be three.